sdk/metricsadvisor/azure-ai-metricsadvisor-perf/src/main/java/com/azure/ai/metricsadvisor/perf/AnomaliesListTest.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.metricsadvisor.perf;
+
+import com.azure.ai.metricsadvisor.models.ListAnomaliesAlertedOptions;
+import com.azure.core.util.Context;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+/**
+ * Performs anomalies list operations.
+ */
+public class AnomaliesListTest extends MetricsAdvisorTestBase<PerfStressOptions> {
+    private static final int MAX_LIST_ELEMENTS = 10;
+
+    /**
+     * Creates AnomaliesListTest object.
+     *
+     * @param options the configurable options for perf testing this class.
+     */
+    public AnomaliesListTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public void run() {
+        super.metricsAdvisorClient
+            .listAnomaliesForAlert(super.alertConfigId,
+                super.alertId,
+                new ListAnomaliesAlertedOptions().setTop(MAX_LIST_ELEMENTS),
+                Context.NONE)
+            .forEach(anomaly -> {
+            });
```

Comment:
Should we need to assert anything on this foreach?

Reply:
I think we just have to ensure all the items are consumed, in case multi-page involved, so a foreach will do. (see the azure-storage blob list perf test)